### ISSUE TITLE
Display Scaling Broken for External Monitors

### ISSUE THREAD

**Issue by funnylookinhat** [pop-os/pop#128]
```
NAME="Pop!_OS"
VERSION="17.10 (Artful Aardvark)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Pop!_OS 17.10 (Artful Aardvark)"
VERSION_ID="17.10"
HOME_URL="https://system76.com/pop"
SUPPORT_URL="http://support.system76.com/"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=artful
UBUNTU_CODENAME=artful
```

**Issue/Bug Description**
When trying to change the scale of external monitors, the "Apply" button never shows up.

**Steps to reproduce (if you know)**
The default for an external 27" 2560x1440 display is "200%" - it should be 100%.  Clicking 100% won't show the "Apply" button though.  Changing the built-in LCD display to 100% shows the allow button, but then you can only have "100%" on BOTH displays, which makes the built-in LCD far too small, but the external looks good.

1) Plug in external monitor.
2) Open "Displays" and select the external monitor.
3) Click "100%" for scale.
4) Notice that "Apply" doesn't show up.  (See attached screenshots below).

**Expected behavior**
Apply should show up - you should be able to use different scales on separate monitors.

**Other Notes**
Relevant Screenshots:

Open the application and select external monitor:
![200](https://user-images.githubusercontent.com/397918/35634286-2d03661e-0660-11e8-9f38-136137d4f572.png)

Select "100%" - Notice that apply is unavailable.
![100](https://user-images.githubusercontent.com/397918/35634294-324958c2-0660-11e8-8430-61ec7f2e88b9.png)

Select the built-in LCD and click "100%" - see that Apply is available.
![100-built-in](https://user-images.githubusercontent.com/397918/35634324-42e7ec02-0660-11e8-849d-3a3b752f180e.png)

**Comment by cassidyjames** [comment]
Yeah… I know this settings pane is buggy in X even upstream. I don't know if there are specific issues filed upstream for it, though.

**Comment by sramkrishna** [comment]
There is an upstream bug that was filed about three  months ago - https://bugzilla.gnome.org/show_bug.cgi?id=788054

**Comment by funnylookinhat** [comment]
Ooops - accidentally closed the ticket...  was going to say that hopefully Ubuntu's pullback to Xorg will force the gnome team to put a bit more effort towards fixing this since it's specific to Xorg sessions.

**Comment by sramkrishna** [comment]
Possibly, but unlikely.  They are focused on the wayland session, and so both Pop!_OS and Ubuntu will have to pick up the slack until wayland is stable enough to make the switch.  I think once Nvidia is onboard that switch will happen quickly.
